packages/client/src/encryption/PublisherKeyExchange.ts

```diff
@@ -106,7 +108,7 @@ export class PublisherKeyExchange implements Context {
                     errorMessage: error.message,
                     groupKeyIds
                 })
-                return await this.keyExchangeStream.response(subscriberId, response)
+                return this.keyExchangeStream.response(subscriberId, response)
```

Comment:
I think `await` is necessary for the `catch`-clause to be called when the promise rejects.